tests/test_xblock_services.py

```diff
@@ -61,7 +61,6 @@ class TestFieldData(XblockTests):
     Tests for the FieldData subclass.
     """
 
-    @api.override_enabled()
```

Comment:
Why the switch of the override from this test to the other?

Reply:
`api.override_enabled` was an obsolete method that enabled a waffle flag that no code actually checked. I removed it in this PR and bumped the version because of that.

The other method (transform test) in this file got a patch because I'm now changing relative date support to be off by default, and that test needs it.